dramatiq/brokers/redis.py

```diff
@@ -273,9 +273,13 @@ def __init__(self, broker, queue_name, prefetch, timeout):
         self.timeout = timeout
 
         self.message_cache = []
-        self.message_refc = 0
+        self.queued_message_ids = set()
         self.misses = 0
 
+    @property
+    def message_refc(self):
```

Comment:
This property is so I can use the same test code with this version of the file and with the previous version of the file.